adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapter.java

```diff
@@ -101,7 +102,7 @@
     private MqttServer insecureServer;
     private AuthHandler<MqttContext> authHandler;
     private final BiConsumer<CommandSubscription, CommandContext> afterCommandPubAckedConsumer = this::afterCommandPublished;
-    private int connectionLimit = Integer.MAX_VALUE;
+    private ConnectionLimitManager connectionLimitManager;
```

Comment:
what about pulling up this property to `AbstractProtocolAdapterBase` and providing a setter for it as well? That would make testing the enforcing of the limit in the adapters much easier ...

Reply:
Done.